meson_options.txt

```diff
@@ -0,0 +1 @@
+option('enable-app', type: 'boolean', value: false, description: 'Enable CUI app')
```

Comment:
It kinda feels like the defaults should be the other way around: nethogs is primarily a CUI tool, and having it as a library is a somewhat experimental edge case.

Reply:
Now, I have added both options (I will be using the library)